src/components/Payment/BraintreeDropInInterface.vue

```diff
@@ -83,6 +83,37 @@ export default {
 			updatingPaymentWrapper: false,
 		};
 	},
+	computed: {
+		formattedAmount() {
+			return numeral(this.amount).format('0.00');
+		},
+		applePaymentRequest() {
+			return {
+				total: {
+					label: 'Kiva',
+					amount: this.formattedAmount,
+				},
+				requiredBillingContactFields: ['postalAddress']
+			};
+		},
+		googleTransactionInfo() {
+			return {
+				totalPriceStatus: 'FINAL',
+				totalPrice: this.formattedAmount,
+				currencyCode: 'USD',
+			};
+		}
+	},
+	watch: {
+		amount(next, prev) {
+			if (next !== prev) {
+				this.btDropinInstance.updateConfiguration('paypal', 'amount', this.formattedAmount);
+				// eslint-disable-next-line max-len
+				// this.btDropinInstance.updateConfiguration('googlePay', 'transactionInfo', this.googleTransactionInfo);
+				// this.btDropinInstance.updateConfiguration('applePay', 'paymentRequest', this.applePaymentRequest);
```

Comment:
Uncommenting these locally didn't result in any errors but I'm unsure of how we'll test them. @eddieferrer did you help with this originally? Do you know if we can verify them in DEV?

Reply:
Ok, it looks like we can at least make sure both Google and Apple initialize correctly once we're in dev. So, I've uncommented these too.

Reply:
@mcstover instructions for testing the complete flow for Google pay and applePay can be found here: https://kiva.atlassian.net/wiki/x/3QA9Kg